RELEASE.md

```diff
@@ -1,78 +1,234 @@
+        - Add the titles of the PRs merged after previous release
+
+### d. Edit the values.yaml
+  - [Fields to edit in values.yaml](https://github.com/kubernetes/ingress-nginx/blob/main/charts/ingress-nginx/values.yaml)
+
+    - tag
+    - digest
+
+### e. Edit the static manifests
+
+  - Prepare to use a script to update the edit the static manifests and set the "image", "digest", "version" etc. fields to the desired value.
+
+
+  - This script depends on python and a specific python package `pip3 install ruamel.yaml`
+
+  - Execute the script to update static manifests using that script [generate-deploy-scripts.sh](https://github.com/kubernetes/ingress-nginx/blob/main/hack/generate-deploy-scripts.sh)
+  - Open some of the manifests and check if the script worked properly
+
+  - Use grep -ir to search for any misses by the script or undesired changes
+
+  - The script should properly set the image and the digest fields to the desired tag and semver
+
+  - Manually fix one problem that the script can not take care of.
+
+    - This problem is wrong formatting of a snippet in the file [deploy-tls-termination.yaml](https://github.com/kubernetes/ingress-nginx/blob/main/deploy/static/provider/aws/deploy-tls-termination.yaml)
+    - In the configMap section, for the configMap named ingress-nginx-controller, the "configMap.data" spec has a snippet
+
+    - This snippet becomes a single line, formatted with the newline character "\n"
 
-    * [test-runner](https://github.com/kubernetes/ingress-nginx/tree/main/images/echo)
-    * [echo](https://github.com/kubernetes/ingress-nginx/tree/main/images/echo)
-    * [cfssl](https://github.com/kubernetes/ingress-nginx/tree/main/images/cfssl)
-    * [fastcgi-helloserver](https://github.com/kubernetes/ingress-nginx/tree/main/images/fastcgi-helloserver)
-    * [httpbin](https://github.com/kubernetes/ingress-nginx/tree/main/images/httpbin)
+    - That single line formatted with "\n" needs to be changed as it does not meet yaml requirements
 
-* Open pull request
-* Merge
-* Wait for [cloud build](https://console.cloud.google.com/cloud-build/builds?project=k8s-staging-ingress-nginx)
+    - At the time of writing this doc, the 'configMap.data' spec is at line number 39.
 
-2a. Promote images:
+    - So editing begins at line 40 (at the time of writing this doc)
 
-* Open pull request to promote [staging image](https://github.com/kubernetes/k8s.io/blob/main/k8s.gcr.io/images/k8s-staging-ingress-nginx/images.yaml)
-    * e2e-test-runner
-    * e2e-test-cfssl
-    * e2e-test-echo
-    * e2e-test-fastcgi-helloserver
-    * e2e-test-httpbin
+    - Make that snippet look like this ;
+      ```
+      data:
+        http-snippet:|
+          server{
+          listen 2443;
+          return 308 https://$host$request_uri;
+          }
+      ```
 
-3. Update references to e2e-test-runner image:
+### f. Edit the changelog
+  [Changelog.md](https://github.com/kubernetes/ingress-nginx/blob/main/Changelog.md) 
```

Comment:
`git log controller-v0.48.1..HEAD --pretty=%s`  as an example of how to get to get the changelog.

Reply:
Documented